storage/client_test.go

```diff
@@ -143,6 +144,9 @@ func (m *multiTestContext) Start(t *testing.T, numStores int) {
 	if m.transport == nil {
 		m.transport = multiraft.NewLocalRPCTransport(m.clientStopper)
 	}
+	if m.storePool == nil {
+		m.storePool = storage.NewStorePool(m.gossip, storage.DefaultTimeUntilStoreDead, m.clientStopper)
```

Comment:
The test one may cause some of the stores to be marked as dead.  We're not testing that here and it can cause other failures.